integration-tests/pkg/mock_sensor/expect_conn.go

```diff
@@ -33,7 +33,7 @@ loop:
 		case <-timer:
 			// we know they don't match at this point, but by using
 			// ElementsMatch we get much better logging about the differences
-			return assert.ElementsMatch(t, expected, s.Connections(containerID), "timed out waiting for networks")
+			return assert.ElementsMatch(t, expected, s.Connections(containerID), "timed out waiting for network connections")
```

Comment:
The changes in this file were made here https://github.com/stackrox/collector/pull/1902/files